sesearch

```diff
@@ -93,6 +93,9 @@ opts.add_argument("-ep", action="store_true", dest="perms_equal",
 opts.add_argument("-ex", action="store_true", dest="xperms_equal",
                   help="Match extended permission set exactly instead of matching any listed "
                   "permission.")
+opts.add_argument("-sp", action="store_true", dest="perms_subset",
```

Comment:
I'd rather have something else as the option as `-sp` is like the source option.

Reply:
What about -ap for "all permissions"?

Reply:
Nevermind, that's confusing because it could also stand for "any permissions"...

Trying to come up with alternatives...

Reply:
What do you think about -fp for "full perms"?

Reply:
How about `-Sp`.

Reply:
-Sp works for me

Reply:
or `-up` or `-Up`